cmd/kubeadm/app/master/discovery.go

```diff
@@ -116,7 +116,8 @@ func newKubeDiscovery(cfg *kubeadmapi.MasterConfiguration, caCert *x509.Certific
 	}
 
 	SetMasterTaintTolerations(&kd.Deployment.Spec.Template.ObjectMeta)
-	SetNodeAffinity(&kd.Deployment.Spec.Template.ObjectMeta, MasterNodeAffinity(), NativeArchitectureNodeAffinity())
+	// TODO: This does not make sense, deployments don't have "affinity"
```

Comment:
Aaah ok. Yeah that was bad.
Can you please fix the comment to make it more explicit what was wrong?